controllers/baremetal_config.go

```diff
@@ -0,0 +1,84 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	metal3iov1alpha1 "github.com/openshift/cluster-baremetal-operator/api/v1alpha1"
+)
+
+const (
+	provisioningNetworkManaged   = "Managed"
```

Comment:
@dhellmann I added enum values for ProvisioningNetwork in commit sadasu@de28b9f. As expected, this resulted in a change in the CRD. I am not sure if this will cause any issues during upgrade from the CRD in MAO in 4.6 to this CRD in 4.7.

Reply:
Maybe for now we should just define them as string constants, then? Eventually we'll have to update that CRD, and we can start making a list of the improvements like this that we want to include when the time comes.